base/linalg/lapack.jl

```diff
@@ -1700,8 +1700,8 @@ for (f, elty) in ((:dggsvd3_, :Float64),
                 throw(DimensionMismatch("B has second dimension $(size(B,2)) but needs $n"))
             end
             p = size(B, 1)
-            k = Ref{BlasInt}()
-            l = Ref{BlasInt}()
+            k = Array(BlasInt, 1)
+            l = Array(BlasInt, 1)
```

Comment:
Yes, they were fine, but I realized all other calls used an array, so better be consistent. But indeed I could have changed `[]` to `[1]` too.